src/components/FeatureViews/Accounts/TabComponents/InstructorSchedule.js

```diff
@@ -67,23 +90,23 @@ const Schedule = ({instructorID}) => {
                 {hoursWorked} hour(s) worked this month
             </h3>
             <FullCalendar
-                allDaySlot={false}
+                allDaySlot={allDayOOO}
                 businessHours={instructorBusinessHours}
                 columnHeaderFormat={{"weekday": "short"}}
                 defaultView="timeGridWeek"
-                events={fullCalendarSessions}
+                events={[...teachingSessions, ...OOO]}
```

Comment:
do you have to put the OOO events in to display them in the allDaySlot? because i'm getting this where the OOO events are taking up the entire day under the all day slots. 
<img width="1521" alt="Screen Shot 2020-03-01 at 3 57 53 PM" src="https://user-images.githubusercontent.com/12959959/75636770-77d81400-5bd6-11ea-8db1-16f8aea1923f.png">

Reply:
Weird, what exactly is the data on your OOO there?